tf-modules/cumulus-rds-tf/main.tf

```diff
@@ -84,7 +85,7 @@ resource "aws_rds_cluster_parameter_group" "rds_cluster_group_v13" {
 resource "aws_rds_cluster" "cumulus" {
   depends_on              = [aws_db_subnet_group.default, aws_rds_cluster_parameter_group.rds_cluster_group]
   cluster_identifier      = var.cluster_identifier
-  engine_mode             = "serverless"
+  engine_mode             = "provisioned"
```

Comment:
Do we have an ADR or something for the change from serverless to provisioned? Just looking for context as it seems like a significant change.

Reply:
@npauzenga  this is a functional change in deploying serverless v2 because of how AWS implemented it.  It's more than a bit confusing, but the doc for the provider shows what they're getting at: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_cluster

Reply:
As Jonathan referenced in the Terraform documentation, we are using serverless v2, but the engine_mode = provisioned is required, albeit a bit misleading.

Where is a good place to add internal documentation?

Reply:
Huh. That is a little surprising but Jonathan's link does clear say that we need to use a provisioned engine mode and specify the instance count for Serverless V2.

I don't think we need to add any Cumulus documentation for this as it's specific to how AWS works. I was mostly just looking for an explanation _somewhere_.